### ISSUE TITLE
Highlight card links have insufficient contrast ratio

### ISSUE THREAD

**Issue by chosak** [cfpb/design-system#1300]
**Describe the bug**

Cards page fails Lighthouse accessibility check due to insufficient contrast ratio.

Lighthouse reports this error: "Background and foreground colors do not have a sufficient contrast ratio." See most recent nightly Lighthouse run [here](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1632356824126-31055.report.html), as run by GitHub Actions [here](https://github.com/cfpb/design-system/runs/3681684398?check_suite_focus=true).

Foreground color is #0072ce. Background color is #e2efd8. Contrast ratio is 4.08. WCAG 2.0 AA requires a contrast ratio of 4.5.

**To Reproduce**

1. Go to https://cfpb.github.io/design-system/patterns/cards in Chrome.
2. Run a Lighthouse accessibility test.
3. See error.

**Expected behavior**

WCAG 2.0 AA requires a contrast ratio of 4.5.

**Screenshots**

Current appearance of highlight card pattern:

![image](https://user-images.githubusercontent.com/654645/134533866-289dd656-3f27-480d-b6f4-454266c42c71.png)

Error as reported in Lighthouse:

![image](https://user-images.githubusercontent.com/654645/134532624-98100536-893b-41d8-9a8e-44df3ddd2706.png)
